pkg/config/agent/config.go

```diff
@@ -71,6 +71,8 @@ type AgentConfig struct {
 	// - gre
 	// - stt
 	TunnelType string `yaml:"tunnelType,omitempty"`
+	// TunnelPort is the port for the TrafficEncapMode Tunnel.
```

Comment:
```suggestion
	// TunnelPort is the destination port for UDP and TCP based tunnel protocols (Geneve, VXLAN, and STT).
	// If unset, it will use the assigned IANA port for the protocol, i.e. 6081 for Geneve, 4789 for VXLAN, and 7471 for STT.
```

Reply:
Thanks for the review, updated.